rfd/0223-install-docs.md

```diff
@@ -0,0 +1,194 @@
+
+A Teleport user may need to mix and match instructions for their use case.
+For example, they may need to install a self-hosted cluster on GKE, deploy an agent into EKS, and then configure the agent to allow App access.
+
+### Proposed Organization
+
+The following organization is proposed, nested under a top-level Platform section:
+
+1. Installation - New page, maps each use case for Teleport to the correct component
+   1. Installing Teleport Agents
+   2. Installing Teleport Client Tools
+   3. Installing Self-Hosted Teleport Clusters
+   4. Installing Teleport Plugins & Integrations
+2. Upgrading - New page, explains the relevance and importance of upgrading each component
+   1. Upgrading Teleport Agents
+   2. Upgrading Teleport Client Tools
+   3. Upgrading Self-Hosted Teleport Clusters
+   4. Upgrading Teleport Plugins & Integrations
+3. Operations - New page, explains what it means to operate Teleport (backups, troubleshooting, speciality config like KMS, multi-region, etc.)
+   1. Operating Teleport Agents
+   2. Operating Teleport Client Tools
+   3. Operating Self-Hosted Teleport Clusters
+   4. Operating Teleport Plugins & Integrations
+
+Notably, cluster operations that are not handled by Cloud should be included under the marketing-branded sections, not Platform.
+For example, a guide to configuring a cluster to support AWS KMS would be included in Operating Self-Hosted Teleport Clusters, while
+a guide for getting started with Teleport roles would be included in the "Zero Trust Access" section.
+The only exception to this rule is global, shared configuration that is directly related to platform operations and does not
+fit into the marketing-branded sections. For example, some of the `cluster_*` or `autoupdate_*` resources may be described in Platform.
+These are generally optional for Cloud users to configure.
+
+Installation instructions should not be complete user guides that include in-product configuration.
+Installation instructions should link to user guides when they are more appropriate, and user guides should link to
+(or import) installation instructions where needed.
+
+### Proposed Action Plan
+
+The proposed action plan is iterative.
+It does not involve a large re-write of any existing documentation, and relies on links between sections where appropriate.
+
+Note: these are best understood by opening https://goteleport.com/docs and following from the top.
+
+- Create new sections and new pages described above.
+- For each existing installation page, separate agent, client, and cluster sections into new pages. No single guide or instruction page should address more than one.
+- "Installation -> Installing Teleport Agents" contains a link to "Enrolling Resources" with separate "Custom Agent Installation" instructions
+- "Introduction -> Installation" moves to "Platform -> Installation"
+- "Introduction -> Upgrading" moves to "Platform -> Upgrading"
+- "Introduction -> Migrate Teleport Plans" moves to "User Guides"
```

Comment:
This doesn't make sense to me. User guides are perhaps not entirely clearly labeled, but they're meant to be for _end_ users who need to access resources, rather than for admins who manage a cluster. Putting a migration plan in here mixes it with access guides, which doesn't seem right to me.

Reply:
I think migration info should live under the platform umbrella.